docs/developer/binarywheels.rst

```diff
@@ -0,0 +1,94 @@
+Building binary wheels
+======================
+
+Several packages in the Zope Foundation repositories contain C code, which
+means they should be packaged and published as binary wheels alongside the
+standard egg archive. Building binary wheels has its pitfalls, though. If not
+done right they can cause application instability and crashes.
+
+
+.. warning::
+    Publishing binary wheels built in a local build environment is highly
+    discouraged! Every build environment is different, binary wheels made for
+    publication on PyPI should only be built in standardized and predictable
+    environments.
+
+To minimize problems we have automated binary wheel building using CI providers
+like Appveyor and GitHub Actions. Wheels are built and uploaded using a special
+GitHub account named ``zope.wheelbuilder``. These configurations and the PyPI
+account are currently maintained by Marius Gedminas, Michael Howitz and Jens
+Vagelpohl. You can contact them by posting your question or bug report in the
+`meta repository issue tracker
+<https://github.com/zopefoundation/meta/issues>`_. Here is a short description
+of the process, please note that some steps can only be performed by the
+configuration maintainers:
+
+Prerequisites
+-------------
+- every project publishing binary wheels must add the PyPI account
+  ``zope.wheelbuilder`` to the list of project maintainers on PyPI. 
+- log into the ``zope.wheelbuilder`` account and create an API token for the
+  project on the `Account settings` page with upload permissions and the
+  project name as scope. Copy the token value - this is the only time you can.
```

Comment:
```suggestion
  project name as scope. Copy the token value |mdash| this is the only time you can.
```

Assuming you want the typographical em-dash there — the one that looks like that and is used as a separator much like a comma.

You then need to add one of two things to the top of the document. Adding:
```rest
.. include:: <isopub.txt>
```

Gets you [definitions for many standard unicode characters](https://docutils.sourceforge.io/docutils/parsers/rst/include/isopub.txt) you can use like that.

Or you can just add the definition yourself:
```rest
.. |mdash|  unicode:: U+02014 .. EM DASH
```

Reply:
I just want a normal dash.